src/status_im/ui/screens/discover/styles.cljs

```diff
@@ -188,4 +190,38 @@
    :margin-top       2
    :margin-bottom    4
    :margin-right     2
-   :background-color :white})
\ No newline at end of file
+   :background-color :white})
+
+(def public-chats-item-container
+  {:flex-direction :row
+   :padding        3})
+
+(def public-chats-icon-container
+  {:flex 0.15})
+
+(defn public-chats-icon [color]
+  {:width            50
+   :height           50
+   :border-radius    25
+   :background-color color
+   :flex-direction   :row
+   :justify-content  :center
+   :align-items      :center})
+
+(def public-chats-icon-text
+  {:font-size 25
+   :color     :gray})
+
+(def public-chats-item-inner
+  {:flex           0.85
```

Comment:
If this reflects the previous `0.15` it would be great to make this explicit (or even make one depend on the other)

Reply:
It does. Ok, that makes sense.